techniques/system/common/1.0/rudder-groups.st

```diff
@@ -0,0 +1,42 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# This file contains all the groups where the node belongs to. The groups are available on two ways:
+# 1/ two classes are added for each groups: 
+#    - group_UUID with UUID replaced by the group unique identifier
+#    - group_NAME with NAME replaced by the transliteration of the group name to ascci, and the 
+#      normalisation of the result
+# 2/ an array "by_uuid" of UUID -> ORIG_NAME, available as a global variable (for example for processing
+#    all node groups to use in templates)
+#  
+#  For example, if node belongs to group: (id: 64f85ba8-39c7-418a-a099-24c2c2909dfd ; name: "Serveurs pré-prod")
+#  we will have the following classes:
+#  - group_64f85ba8_39c7_418a_a099_24c2c2909dfd
+#  - group_Serveur_pre_prod
```

Comment:
According to the example above this is missing a "s" (group_Serveurs_pre_prod).

CFEngine class names are case-sensitive... Maybe we should just lower-case the name aswell to avoid confusion?